test/unit/full_text_search/similar_search_issue_test.rb

```diff
@@ -0,0 +1,119 @@
+require File.expand_path("../../../test_helper", __FILE__)
+
+module FullTextSearch
+  class SimilarSearchIssueTest < ActiveSupport::TestCase
+    include ActiveJob::TestHelper
+
+    fixtures :roles
+    fixtures :users
+
+    def setup
+      IssueContent.destroy_all
+      User.current = User.find(1)
+      @project = Project.generate!
+      User.add_to_project(User.current, @project)
+    end
+
+    def test_same_structure_on_issue
+      fts_engine_groonga_open_source =
+        perform_enqueued_jobs(only: FullTextSearch::UpdateIssueContentJob) do
+          Issue.generate!(
+            project: @project,
+            subject: "ぐるんが",
+            description: "高速に検索できます。 オープンソースです。")
+        end
+      fts_engine_pgroonga_open_source =
+        perform_enqueued_jobs(only: FullTextSearch::UpdateIssueContentJob) do
+          Issue.generate!(
+            project: @project,
+            subject: "ぴーじーるんが",
+            description: "PostgreSQLに組み込んで高速に検索できます。 オープンソースです。")
+        end
+
+      similar_issues =
+        fts_engine_groonga_open_source.similar_issues(
+          project_ids: [@project.id])
+      assert_equal([fts_engine_pgroonga_open_source],
+                   similar_issues)
+
+      perform_enqueued_jobs(only: FullTextSearch::UpdateIssueContentJob) do
+        fts_engine_groonga_open_source.update!(description: nil)
+      end
+      similar_issues =
+        fts_engine_groonga_open_source.similar_issues(
+          project_ids: [@project.id])
+      assert_equal([], similar_issues)
```

Comment:
There are two tests, one with and one without `similar_issues`.
Since the points of the tests are different, why don't we separate them?

Reply:
I wanted to verify that once a record appeared in the search results and also no longer appears when the record is deleted. That's why I combined both checks in a single test.